docs/rewind/manual/licensing.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 9
+---
+
+# Licensing
+
+## Overview
+
+Rewind is available in three different editions, which differ only in the number of channels they
+can process:
+
+| Edition | Channels | RTA |
+|---------|----------|-----|
+| Personal | 16 | No |
+| Professional | 64 | Yes |
+| Production | 128 (wow!) | Yes |
+
+To protect your investment, guard against intellectual property theft, and safeguard our
+R&D for the future, we use the *iLok* dongle system to license Rewind. You will need an iLok 2 or
+iLok 3 to run *Rewind*.
+
+We chose *iLok* because we know many engineers will have one already, but if you don't, they can be
+obtained cheaply from your favourite rainforest-related eCommerce provider, or alternatively
+*Fourier Audio* can provide pre-licensed dongles ready for use.
+
+The first time you start *Fourier Rewind*, you'll be prompted to insert your *iLok* and
+enter your activation code to obtain your licence. After that, you should never need to think about
+it ever again; the licence simply lives on your *iLok*.
+
+All three editions run via the same *Rewind* application; if you find you need more channels, simply
+upgrade your licence - mid-gig if necessary! Upgrade pricing [is
+available](mailto:hello@fourieraudio.com).
+
+## iLok Login Details
+
+You will be required to have an iLok account linked to your physical iLok.
+
+If this is your first time using an iLok, you can create an iLok account on PACE's website [here](https://www.ilok.com/#!registration).
+
+During account creation, you will create a User ID - this is the User ID that allows us to deliver your Rewind licence to your account.
+
+Forgotten your User ID? Visit [PACE Support](https://www.ilok.com/#!support/help/102029545) for further assistance.
+
+## Activating Your Rewind Licence
+
+When you begin a Rewind trial, or purchase a Rewind licence, you will be asked for your User ID. This will enable us to deposit your licence magically into your iLok account, ready for activation onto your physical iLok.
+
+Once deposited to your account, in order to activate your licence:
+1. Launch iLok License Manager (installed with Rewind if you don't already have it)
+2. Login to your iLok account
```

Comment:
```suggestion
2. Log in to your iLok account
```

"Login" is a noun, "log in" is the action.

We need to scrap the entire English language and start over from scratch. I'll file an issue.